admin_guide/backup_restore.adoc

```diff
@@ -0,0 +1,122 @@
+= Backup and Restore
```

Comment:
I think a lot of the "backup"s need to be "back up":

http://grammarist.com/usage/back-up-backup/